src/annotator/anchoring/range.js

```diff
@@ -423,17 +440,15 @@ export class SerializedRange {
     // Node.compareDocumentPosition() to decide when to set the
     // commonAncestorContainer and bail out.
 
-    const contains = (a, b) => a.compareDocumentPosition(b) & 16;
-    $(range.startContainer)
-      .parents()
-      .each(function () {
-        if (contains(this, range.endContainer)) {
-          range.commonAncestorContainer = this;
-          // bail out of loop
-          return false;
-        }
-        return true;
-      });
+    const contains = (a, b) =>
+      a.compareDocumentPosition(b) & Node.DOCUMENT_POSITION_CONTAINED_BY;
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/API/Node/compareDocumentPosition for enum names and associated values.